src/argilla/client/feedback/schemas/records.py

```diff
@@ -19,6 +19,7 @@
 from pydantic import BaseModel, Extra, Field, PrivateAttr, StrictInt, StrictStr, conint, validator
 
 from argilla.client.feedback.schemas.enums import ResponseStatus
+from argilla.server.enums import RecordSortField, SortOrder
```

Comment:
We shouldn't be importing stuff from the `argilla.server` so I guess you meant

```suggestion
from argilla.client.feedback.schemas.enums import RecordSortField, SortOrder
```